library/src/main/java/com/schibsted/spain/barista/internal/matcher/SeekBarAssertions.kt

```diff
@@ -0,0 +1,60 @@
+package com.schibsted.spain.barista.internal.matcher
+
+import android.support.test.espresso.AmbiguousViewMatcherException
+import android.support.test.espresso.Espresso
+import android.support.test.espresso.NoMatchingViewException
+import android.support.test.espresso.ViewAssertion
+import android.support.test.espresso.assertion.ViewAssertions
+import android.support.test.espresso.matcher.ViewMatchers
+import android.view.View
+import android.widget.SeekBar
+import com.schibsted.spain.barista.internal.failurehandler.SpyFailureHandler
+import com.schibsted.spain.barista.internal.failurehandler.description
+import org.hamcrest.Matcher
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.`is`
+
+object SeekBarAssertions {
```

Comment:
Random thought: 
Progress is actually not part of `SeekBar` but `ProgressBar` which is the parent class. Does it make sense that the assertions are also compatible with `ProgressBar`? What do you think?

The interaction is similar, but in that case since the user can't change the progress via the UI, I think it's better to limit it to the SeekBar.

Reply:
I didn't think about it, but I do agree that there is no point on having assertions for a view whose progress the dev has no control. I guess that if int the future someone finds a case where it is needed adding support shouldn't be complicated.